plugins/search-react/src/components/SearchBar/SearchBar.tsx

```diff
@@ -80,7 +80,8 @@ export const SearchBarBase: ForwardRefExoticComponent<SearchBarBaseProps> =
         value: defaultValue,
         label,
         placeholder,
-        inputProps: defaultInputProps = {},
+        inputProps = {},
+        InputProps = {},
```

Comment:
do I understand this correctly that the `InputProps` is just for `endAdornment`? If so would actually recommend to change this to something like `endAdornmentInputProps` to make it clear what its used for when using the component. What do you think?

Reply:
Will do.

Reply:
Commit amended, extended the example on `Custom Style` use case, showing how to use `InputProps` to override Material UI Styles.